### ISSUE TITLE
Combine documentation from foam-template; previously: "Document Alt+C to toggle checkboxes"

### ISSUE THREAD

**Issue by infogulch** [foambubble/foam#1025]
### Is your feature request related to a problem? Please describe.

It requires unnecessary finicky and precise editing to change a checkbox state in editor view. This should be very easy because it happens often.

Luckily, a solution to such a problem is already implemented! However its buried in the [vscode-markdown's CHANGELOG.md](https://github.com/yzhang-gh/vscode-markdown/blob/master/CHANGELOG.md#0110-20171018)

### Describe the solution you'd like

Document that this is a keyboard shortcut that would be useful for todo / task management workloads. Perhaps in recipes or [keybindings](https://github.com/foambubble/foam/blob/master/docs/features/key-bindings.md)

### Describe alternatives you've considered

This shortcut is not very discoverable, so the alternative is making each user spend time searching for this feature on their own.

**Comment by riccardoferretti** [comment]
This is documented here: https://github.com/foambubble/foam-template/blob/master/docs/how-to/shortcut-list.md

I agree it might be helpful to link the documentation in this repo as well though, I agree with your suggestion about using the `keybindings` file.

**Comment by riccardoferretti** [comment]
The template is just a way to kickstart a project. It’s not necessary.
As part of the onboarding of the template we have added a “walkthrough” of the main features.
But the real source of truth is really the docs folder in the foam repo

**Comment by riccardoferretti** [comment]
I think you are totally right. 
I agree that the foam repo should be the source of truth, and I agree that it would be good to separate user docs from dev docs. 
The only thing I can see being a bit challenging is what to put in foam-template, given these two conflicting requirements:
- it is not the source of truth, as that is foam/docs
- it needs to have enough info to onboard users and help them get started with the system 

Maybe a way to "sync" the user side of the docs, whether manually  or not, might be the answer, but I am not sure.. thoughts?

**Comment by riccardoferretti** [comment]
Yup I think that makes sense.
Would you like to take a stab at it? I will be happy to support.
Otherwise we'll wait for someone to pick this up :)

**Comment by infogulch** [comment]
Sure I'll see what I can do this weekend.

**Comment by infogulch** [comment]
I created a draft PR with some of the changes I mentioned. I did some editing / rewording so the docs can work across the main website and also in foam-template. There are several parts that are still just unfinished, and some things that I didn't even try to do yet like sync link reference definitions or make sure the website can still generate, but I thought I'd post my progress in case you have any thoughts.
